custom_components/localtuya/translations/pt.json

```diff
@@ -0,0 +1,190 @@
+{
+    "config": {
+        "abort": {
+            "already_configured": "O dispositivo já foi configurado.",
+            "device_updated": "A configuração do dispositivo foi atualizada!"
+        },
+        "error": {
+            "authentication_failed": "Falha ao autenticar.\n{msg}",
+            "cannot_connect": "Não é possível ligar ao dispositivo. Verifique se o endereço está correto e tente novamente",
+            "device_list_failed": "Falha ao recuperar a lista de dispositivos.\n{msg}",
+            "invalid_auth": "Falha ao autenticar com o dispositivo. Verifique se o ID do dispositivo e a chave local estão corretos.",
+            "unknown": "Ocorreu um erro desconhecido. Consulte o registro para obter detalhes.",
+            "entity_already_configured": "A entidade com este ID já foi configurada.",
+            "address_in_use": "AddresO endereço usado para descoberta já está em uso. Certifique-se de que nenhum outro aplicativo o esteja usando (porta TCP 6668).s used for discovery is already in use. Make sure no other application is using it (TCP port 6668).",
```

Comment:
Message unformatted and not completely translated.

Reply:
Fixed